template/.claude/commands/issue.md

```
@@ -0,0 +1,162 @@
+
+Use the GitHub CLI to fetch issue details:
+
+```bash
+gh issue view [ISSUE_NUMBER] --json title,body,labels,comments,state
+```
+
+If the `gh` CLI is not installed or authenticated, show:
+
+```
+GitHub CLI not available or not authenticated!
+Run: gh auth login
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)